### ISSUE TITLE
Add. 14,665, fol. 28 | https://bl.vuexistapps.us/manuscript/444 (Part15)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1110]
### Problems with Manuscript Overview

- No date is mentioned in the MS overview. Wright doesn't mention any date either.

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- Notice "locus:" in "Add. 17,217,locus: foll. 37 and 38"
- Lack of space after "," in "Add. 17,217,locus: foll. 37 and 38"
- Two period marks in "as Add. 17,217,locus: foll. 37 and 38.."
- The printed catalog reads, "Unfortunately the greater part of one column on each side has been covered with rude Arabic writing, and can therefore only be deciphered with difficulty." In the online version, however, this sentence breaks into two parts, the first part being placed in the Additions section and the second in Phys. Des. So, Phys. Des. only has " It can therefore only be deciphered with difficulty." The word "therefore" in the online version is not clear. In the printed version, the word "therefore" denotes that the text is hard to decipher because a great part of one column is covered by Arabic writings, but in the online version this information cannot be delivered. 
- Hand 1 is empty. Wright doesn't say anything either. 

### Categories of Physical Description Problems

- [X] Whitespace
- [X] Typo(s)
- [X] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
@wlpotter really messy item!

Resolved
- PhysDesc cleaned. "therefore" deleted
- Addition1, "quote" changed "foreign" to avoid line breaks

Unresolved
- Hand 1, Wright does not provide info. Leave blank?
- Subject identification: XML gives grammar and scientific literature. Since Wright doesn't say anything about the content, do we need to double check?
- Many display issues

**Comment by wlpotter** [comment]
Yeah this one is really complicated, especially since Wright leaves a lot implied by his reference to Add. 17,217. I'll assign this to myself to look into it more closely as I think the changes needed here might end up being pretty complex
